eslint.config.mjs

```diff
@@ -89,6 +88,7 @@ export default defineConfig([
 			"@typescript-eslint/unbound-method": "off",
 			"jest/unbound-method": "error",
 			"jest/no-focused-tests": "error",
+			"max-lines": "off",
```

Comment:
This isn't a formal request but a note that now that we're using flat config we can add comments when we modify default rules to provide context for future (e.g. the comment here might be something like "test files are allowed to be long because they're structured and need to be comprehensive")